src/main/EMSError.js

```diff
@@ -61,10 +61,18 @@ const MQ_PROCESS_ERROR_CODES = {
     MQ_ID_DOESNT_EXIST: 152,
 }
 
+// Exams
+const EXAM_PROCESS_ERROR_CODES = {
+    SET_ANSWERS_TO_QUESTION: 200,
+    ADD_QUESTION_TO_EXAM: 201,
+    GET_EXAM_BY_ID: 202,
```

Comment:
I would rather like the name of the error code to explain why the error is throw. 
instead of `GET_EXAM_BY_ID` have it be `INVALID_EXAM_ID` or `EXAM_ID_NOT_FOUND`. something like that...